spacy/language.py

```diff
@@ -265,7 +293,112 @@ def pipe_labels(self):
                 labels[name] = list(pipe.labels)
         return labels
 
-    def get_pipe(self, name):
+    @classmethod
+    def has_factory(cls, name: str) -> bool:
+        """RETURNS (bool): Whether a factory of that name is registered."""
+        return name in registry.factories
+
+    @classmethod
+    def get_factory_meta(cls, name: str) -> "FactoryMeta":
+        """RETURNS (FactoryMeta): The meta for the given factory name."""
+        if name not in cls._pipe_meta:
+            raise ValueError(Errors.E967.format(name=name))
```

Comment:
Should we specify "factory" vs "component" in the error message of E967? Because right now it says "component meta", also when calling`get_factory_meta` ?

Reply:
Ah, good point, yes! 

(I initially had only one dict for factories but then I realised that we also need to be able to fetch meta for any given component _instance_, so I added the second.)